.github/workflows/zap_templates.yaml

```diff
@@ -49,4 +49,5 @@ jobs:
             - name: Check for uncommited changes
               run: |
                 git add .
+                git diff-index HEAD --
```

Comment:
should we do a --name-only here to avoid having too much output? or is the diff content relevant?